commit/chainfee/observation.go

```diff
@@ -0,0 +1,12 @@
+package chainfee
+
+import cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+func (p *processor) ObserveFChain() map[cciptypes.ChainSelector]int {
```

Comment:
Does this really need a separate file?

Reply:
Just for consistency between all commit processors. I was even going to make proxies for ccipReader calls that happens in actual observation but decided against it for simplicity.

Reply:
I would expect `observation.go` to have processor observation logic, but it's in processor.go, which is confussing